src/hooks/useKeyboardShortcut/useKeyboardShortcut.mdx

```diff
@@ -0,0 +1,304 @@
+import { Meta, Stories } from '@storybook/blocks';
+import * as KeyboardShortcutStories from './useKeyboardShortcut.stories';
+
+<Meta of={KeyboardShortcutStories} title="Hooks / useKeyboardShortcut" />
+
+# useKeyboardShortcut
+
+Use this hook to add keyboard shortcuts to your application. Supports single keys, modifiers and
+sequences.
+
+The default configuration works out of the box with inputs and is cross-platform.
+
+```ts
+useKeyboardShortcut('ctrl+a', () => {
+  console.log('shortcut pressed');
+});
+```
+
+## Reference
+
+```ts
+type UseKeyboardShortcutOptions<T extends EventTarget = EventTarget> = {
+  ref?: RefObject<T | null> | T | HTMLElement | null | undefined;
+  ignoreWhenInputFocussed?: 'auto' | true | false;
+  command?: Shortcut;
+  convertPlatforms?: boolean;
+  windowsCommand?: Shortcut;
+  macOsCommand?: Shortcut;
```

Comment:
What about other platforms? The `is-hotkey` package has a custom `mod` identifier that acts the same across platforms, I like that approach.

Reply:
For this I'm auto-converting `cmd` to `ctrl` on windows. However, this means that you cannot use the `windows` key on windows with this behaviour enabled – not sure if you ever want to make use of the windows key in web apps, but still.

Supporting `mod` as a generic shortcut might indeed be a nicer option.